pkg/machine/wsl/config.go

```diff
@@ -1,4 +1,4 @@
-//go:build windows
+//go:build tempoff
```

Comment:
Does this need to be switched somewhere else?

Reply:
this is just enough to get things ignored ... the WSL PR will take care of all this